src/backend/columnar/columnar_writer.c

```diff
@@ -118,7 +118,7 @@ ColumnarBeginWrite(RelFileNode relfilenode,
 															 ALLOCSET_DEFAULT_SIZES);
 
 	bool *columnMaskArray = palloc(columnCount * sizeof(bool));
-	memset(columnMaskArray, true, columnCount);
+	memset(columnMaskArray, true, columnCount * sizeof(bool));
```

Comment:
what about `memset` in `SerializeBoolArray`?

Reply:
it just takes in a strict byte count, so I don't think we need to anything for that

Reply:
oh I guess I got what you meant